### PR TITLE
DBZ-1017 Record schema evolution into history correctly

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1017

### CODE REVIEW

File: debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerDatabaseSchema.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public void applySchemaChange(SchemaChangeEvent schemaChange) {
         tables().overwriteTable(table);
 
         TableChanges tableChanges = null;
-        if (schemaChange.getType() == SchemaChangeEventType.CREATE && schemaChange.isFromSnapshot()) {
+        if (schemaChange.getType() == SchemaChangeEventType.CREATE) {
```
Comment (gunnarmorling) [inline debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerDatabaseSchema.java]: @jpechane, why is it that only CREATE statements are tracked?

Comment (jpechane) [reply debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerDatabaseSchema.java]: @gunnarmorling The fact is that from the implementaiton POV there is no difference between `CREATE` and `ALTER` so we are using a single event.

Comment (gunnarmorling) [reply debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerDatabaseSchema.java]: Ok; I have to admit this still is a bit blurry to me. Would be nice to get some clarity in the context of [DBZ-993](https://issues.jboss.org/browse/DBZ-993).

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks, @jpechane!

**Pull request closed without merging**